releases/release-1.21/release-notes/maps/pr-98730-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 98730
+releasenote:
+  text: Several flags related to the deprecated dockershim which are present in the
+    kubelet command line are now showing warning about the future deprecation.
```

dockershim deprecation already happened, just not removed 😄 deprecated = will be removed in future release

```suggestion
  text: Kubelet command line flags related to dockershim are now showing deprecation 
    message as they will be removed along with dockershim in future release.
```

---

I had wonder how to phrase this one a lot. Thanks, it sounds better than.